ocs_ci/helpers/dr_helpers_ui.py

```diff
@@ -0,0 +1,274 @@
+     Submariner check will not be done for Metro DR and only navigation to ACM console will take place.
+
+    Args:
+        setup_acm_ui: login function to ACM on conftest file
+
+    """
+    acm_obj = AcmPageNavigator(setup_acm_ui)
+    acm_add_clusters_obj = AcmAddClusters(setup_acm_ui)
+    acm_obj.navigate_infrastructure_env_page()
+    # Submariner validation is not applicable for Metro-DR
+    multicluster_mode = config.MULTICLUSTER.get("multicluster_mode", None)
+    if multicluster_mode == "regional-dr":
+        acm_add_clusters_obj.submariner_validation_ui()
+
+
+def check_cluster_status_on_acm_console(
+    setup_acm_ui,
+    down_cluster_name=None,
+    cluster_names=None,
+    timeout=900,
+    expected_text="Ready",
+    wait=False,
+):
+    """
+    This function checks the current status of imported clusters on the ACM console.
+    These clusters are the managed OCP clusters and the ACM Hub cluster.
+
+    Args:
+        setup_acm_ui: login function to ACM on conftest file
+        down_cluster_name (str): If Failover is performed when a cluster goes down, wait is set to True & the updated
+                            status of cluster unavailability is checked on the ACM console.
+                            It takes the cluster name which is down.
+        cluster_names (list): This is a list of cluster names involved in a DR setup. You can either pass the cluster
+                            names as args in the form of list, but if not passed, it fetches the primary & secondary
+                            cluster names passed at run time for context setting
+                            (max. 3 for now including ACM Hub cluster).
+                            ACM Hub cluster name is hard coded as "local-cluster" as the name is constant & isn't
+                            expected to change.
+        timeout (int): Timeout to wait for certain elements to be found on the ACM UI
+        expected_text (str): Any particular string/status of the cluster to be checked on the ACM console.
+        wait (bool): When True, cluster status check will be done for the expected text/status in the given timeout.
+
+    """
+
+    ocp_version = version.get_semantic_ocp_version_from_config()
+    acm_obj = AcmPageNavigator(setup_acm_ui)
+    acm_loc = locators[ocp_version]["acm_page"]
+
+    acm_obj.navigate_clusters_page()
+
+    if down_cluster_name:
+        wait = True
+        check_cluster_unavailability = acm_obj.wait_until_expected_text_is_found(
+            format_locator(acm_loc["cluster_status_check"], down_cluster_name),
+            expected_text="Unavailable",
+            timeout=timeout,
+        )
+        if check_cluster_unavailability:
+            log.info(f"Down cluster {down_cluster_name} is {expected_text}")
+            return
+        else:
+            cluster_status = acm_obj.get_element_text(acm_loc["cluster_status_check"])
+            assert (
+                cluster_status == "Ready"
```

Comment:
please save/use an existing constant for "Ready"

Reply:
Done